packages/components/src/internal/components/ExpandableContainer.tsx

```diff
@@ -63,10 +64,10 @@ export class ExpandableContainer extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { children, iconSrc, iconFaCls, isExpandable, clause, links, iconClickOnly, containerCls } = this.props;
+        const { children, iconSrc, iconFaCls, isExpandable, clause, links, iconClickOnly, containerCls, useGreyTheme } = this.props;
         const { visible, isHover } = this.state;
         const hasOnClick = this.props.onClick !== undefined;
-        const containerDivCls = iconClickOnly ? 'container-expandable-icononly' : 'container-expandable-detail';
+        const containerDivCls = iconClickOnly || useGreyTheme ? 'container-expandable-icononly' : 'container-expandable-detail';
```

Comment:
I'm not seeing a usage for `iconClickOnly` anymore and it seems we may not need two css classes here.  the `container-expandable-icononly` class doesn't change the cursor to pointer except when over the icon, and that's not what we want.  Can we remove the `iconClickOnly` property and update the `container-expandable-icononly` css class name to something like  `container-expandable`?  This will likely lead to a need to update a Selenium test locator somewhere.  (There's opportunity for some refactoring of the css classes for reuse here, but that's not required. )

Reply:
Done! 
`container-expandable` already existed, so I selected `container-expandable-grey`, and to make the logic parallel, I updated `container-expandable-detail` to `container-expandable-blue`. A testAutomation PR is now open, too.